sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch

```diff
@@ -188,4 +188,14 @@
     <node name="arm_group_position_controller_spawner" pkg="controller_manager" type="spawner" output="screen"
           args="ra_trajectory_controller la_trajectory_controller"/>
   </group>
+
+  <arg name="record_hand_data" value="$(optenv SR_RECORD_HAND_DATA disabled)"/>
+  <arg name="log_directory" value="$(optenv HOME)/shadow_bag_storage/sr_record_all_hand_data"/>
```

Comment:
do we need to create this folder in advance?

Reply:
Aurora is handling that